bubblepicker/src/main/java/com/igalata/bubblepicker/physics/CircleBody.kt

```diff
@@ -31,18 +31,18 @@ class CircleBody(val world: World, var position: Vec2, var radius: Float, var in
 
     var isVisible = true
 
-    private val margin = 0.01f
-    private val damping = 25f
     private val shape: CircleShape
         get() = CircleShape().apply {
-            m_radius = radius + margin
+            m_radius = radius
             m_p.setZero()
         }
 
     private val fixture: FixtureDef
         get() = FixtureDef().apply {
-            this.shape = this@CircleBody.shape
-            this.density = this@CircleBody.density
+            shape = this@CircleBody.shape
+            density = this@CircleBody.density
+            restitution = 0.99f
```

Comment:
Same comment as above.
The problem is: we want the bubbles to keep moving indefinitely

Reply:
So yes, with these changes the bubbles do stop. Even when setting a value of 1, the bubbles eventually stop. So the stopping seems to be a result of something else but I haven't been able to figure it out.

Reply:
In that case, do put these to 1 & 0, respectively, and add a TODO regarding needing to figure out where the stop happens

Reply:
However they take a while to stop. I know that was one of the issues raised previously and to try and cheat that: Every time you select a bubble an impulse is given outwards, deselecting sends an impulse inwards. Both of these actions keep the bubbles moving a bit.